source/lib/src/gpu/CMakeLists.txt

```
@@ -43,6 +43,15 @@ if(USE_CUDA_TOOLKIT)
     message(FATAL_ERROR "CUDA version must be >= 9.0")
   endif()
 
+  # CUDA 13.0+ requires C++17
+  if(${CMAKE_CUDA_COMPILER_VERSION} VERSION_GREATER_EQUAL "13.0")
```

It's not recommended to use `${xxx}` inside `if`.
```suggestion
  if(CMAKE_CUDA_COMPILER_VERSION VERSION_GREATER_EQUAL "13.0")
```